hw/mcu/stm/stm32_common/src/hal_spi.c

```diff
@@ -172,6 +172,7 @@ stm32_resolve_spi_irq(SPI_HandleTypeDef *hspi)
 #endif
     default:
         assert(0);
+        return 0;
```

Comment:
returning 0 here, would lead to unpredictable behavior.

Reply:
Will default to SPI1_IRQn